.github/workflows/interop-test.yml

```diff
@@ -0,0 +1,36 @@
+name: Interoperability Testing
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+
+jobs:
+  run-multidim-interop:
+    name: Run multidimensional interoperability tests
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ipfs/aegir/actions/cache-node-modules@master
+        with:
+          directories: |
+            ./interop/node_modules
+      - name: Build interop
+        run: (cd interop && npm i && npm run build)
+      - name: Build images
+        run: (cd interop && make)
+      # - name: Save package-lock.json as artifact
```

Comment:
Why is this commented out? I think that if we want tests to be reproducible for a given commit this would be required otherwise we are vulnerable to dep breakages